models/docs/marts/trade_agg.md

```diff
@@ -0,0 +1,301 @@
+[comment]: < Trade Aggregations -
+
+{% docs trade_agg %}
+
+This table contains the aggregated values for the trades involving different pairs of selling/buying assets, over different time periods in the past.
+
+{% enddocs %}
+
+{% docs trade_day_agg %}
```

Comment:
nit: this column is just `day_agg`

Reply:
The doc name is only relevant for when calling it from the .yml file, so there's no issue with it being different from the actual column name. This was called a specific name to keep the custom description.